internal/painter/gl/shaders.go

```diff
@@ -45,3 +45,37 @@ var shaderSimpleesVert = &fyne.StaticResource{
+}
+var shaderRoundRectangleesFrag = &fyne.StaticResource{
+	StaticName: "round_rectangle_es.frag",
+	StaticContent: []byte(
+		"#version 100\n\n#ifdef GL_ES\n# ifdef GL_FRAGMENT_PRECISION_HIGH\nprecision highp float;\n# else\nprecision mediump float;\n#endif\nprecision mediump int;\nprecision lowp sampler2D;\n#endif\n\n/* scaled params */\nuniform vec4 frame_size;  //frame_size_width = x, frame_size_height = y (z, w NOT USED)\nuniform vec4 rect_coords; //x1 [0], x2 [1], y1 [2], y2 [3]; coords of the rect_frame\nuniform float stroke_width_half;\nuniform vec4 rect_size_half; //rect_size_width_half = x, rect_size_height_half = y (z, w NOT USED)\nuniform float radius;\n/* colors params*/\nuniform vec4 fill_color;\nuniform vec4 stroke_color;\n\n\n\nfloat calc_distance(vec2 p, vec2 b, float r)\n{\n    vec2 d = abs(p) - b + vec2(r);\n\treturn min(max(d.x, d.y), 0.0) + length(max(d, 0.0)) - r;   \n}\n\nvoid main() {\n\n    vec4 frag_rect_coords = vec4(rect_coords[0], rect_coords[1], frame_size.y - rect_coords[3], frame_size.y - rect_coords[2]);\n    // vec2 rect_size_half =  vec2(frag_rect_coords[1] - frag_rect_coords[0], frag_rect_coords[3] - frag_rect_coords[2]) * 0.5 - vec2(stroke_width_half);\n    vec2 vec_centered_pos = (gl_FragCoord.xy - vec2(frag_rect_coords[0] + frag_rect_coords[1], frag_rect_coords[2] + frag_rect_coords[3]) * 0.5);\n\n    float distance = calc_distance(vec_centered_pos, vec2(rect_size_half.x, rect_size_half.y), radius - stroke_width_half);\n\n    vec4 from_color = stroke_color; //Always the border color. If no border, this still should be set\n    vec4 to_color = vec4(1.0, 1.0, 1.0, 0.0); //Outside color\n\n    if (stroke_width_half > 0.0)\n    {\n        if (distance < 0.0)\n        {\n            to_color = fill_color;   \n        } \n        \n        distance = abs(distance) - stroke_width_half;\n    }\n\n    float blend_amount = smoothstep(-1.0, 1.0, distance);\n\n    // final color\n    gl_FragColor = mix(from_color, to_color, blend_amount);\n    //gl_FragColor = vec4(vec3(blend_amount), 1.0);\n    //gl_FragColor = vec4(vec3(abs(distance) / (2.0 * corner)), 1.0);\n\n}\n"),
+}
+
+// rectangle_es.vert is used for rectangle_es.frag and round_rectangle_es.frag
+var shaderRectangleesVert = &fyne.StaticResource{
+	StaticName: "rectangle_es.vert",
+	StaticContent: []byte(
+		"#version 100\r\n\r\n#ifdef GL_ES\r\n# ifdef GL_FRAGMENT_PRECISION_HIGH\r\nprecision highp float;\r\n# else\r\nprecision mediump float;\r\n#endif\r\nprecision mediump int;\r\nprecision lowp sampler2D;\r\n#endif\r\n\r\nattribute vec2 vert;\r\nattribute vec2 normal;\r\n\r\nvoid main() {\r\n    vert;  //Workaround, because WebGL optimizes away attributes unused\r\n    gl_Position = vec4(normal, 0, 1);\r\n}\r\n"),
+}
```

Comment:
I think you need to re-generate this using `go generate` in this directory.
The comments indicate to me that the files has been manually edited afterwards and some variable names do not match compared to what I get when generating the file locally. As it is a generated file, we need to be able to regenerate it without making manual modifications afterwards.

Reply:
I did't know that and used ```fyne bundle``` for the single files.

Reply:
After ```go generate``` of ```shaders.go``` there are a lot "\r".

Reply:
Hmm. You are right. That's strange and not entirely desirable. Are you on Windows by any chance?

Reply:
There are some weird encoding issues on windows consoles. Using the "fyne bundle" with "-o" instead of terminal redirect ">>" works around this.